API/NoDocs/Microsoft.Extensions.DependencyInjection/HttpResilienceHedgingHttpClientBuilderExtensions.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Microsoft.Extensions.Http.Resilience'
+
+using System;
+using Microsoft.Extensions.Http.Resilience;
+using Polly;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class HttpResilienceHedgingHttpClientBuilderExtensions
```

Comment:
Do we need the extra class? Can we move these to `HttpResilienceHttpClientBuilderExtensions`?

Reply:
move to the single class